compose/cli/utils.py

```diff
@@ -120,3 +123,15 @@ def is_mac():
 
 def is_ubuntu():
     return platform.system() == 'Linux' and platform.linux_distribution()[0] == 'Ubuntu'
+
+
+def get_version_info(scope):
+    versioninfo = 'Docker-Compose version: %s' % __version__
```

Comment:
This should either say `docker-compose` or `Docker Compose`. We're not consistent across the platform (Machine uses the first form, Engine the second) but I vote for the first.